[ExplosmBridge] Rewrite to work without feedburner

jNullj opened the pull request:
re-wrote the bridge to scrap from the new explosm site as the old method of using feedburner is not working anymore, feedburner is stuck on dec/22 when the explosm site changed.

fix issue #2415

jNullj committed:
[ExplosmBridge] Rewrite to work without feedburner

re-wrote the bridge to scrap from the new explosm site as the old method of using feedburner is not working anymore, feedburner is stuck on dec/22 when the explosm site changed.

jNullj commented:
I tested this on my own host with latest release and it works as intended (docker on archlinux host, tested with tt-rss as client)

em92 commented:
Ping @Bockiii for reviewing this change.

em92 commented on bridges/ExplosmBridge.php:
```diff
@@ -1,22 +1,58 @@
+				'type' => 'number',
+				'required' => true,
+				'title' => 'Maximum number of items to return',
+				'defaultValue' => 5
```
Do we need "required" here?
This setting will break previous usage, where "/?action=display&bridge=Explosm&format=Html" worked. If I apply this change, users need to reedit their rss feeds.

jNullj replied:
I did not think about it, this is a great insight. I makred it required becouse a limit is required. I could make it so the default is 5 which is the number of posts the bridge returned when it used feedburner before this change. Does the `'defaultValue' => 5` will take care of that, im not sure.

jNullj replied:
Tested myself, i fixed this and tested for old feed urls to work.

em92 reviewed (changes requested):
Hi, @jNullj!
Please fix linting and comment below.

jNullj committed:
Fixed linting

jNullj committed:
Remove post limit requirment

For backwards compatibility with links used by the previous version of this bridge assume default of 1 posts if the parameter is missing.

Bockiii commented:
Wasn't able to test this until now, but it looks like it works well. Thanks @jNullj !
